Bottom: 394fa826e41a3c7be3c82f1eceea3b9dd4fc4355
Top:    76329407775f254d389b1597594ba1b0fcf6a399
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:35 +0000

Update netclass clearances

4.0 -> 4.5
0.2 -> 0.3


---

diff --git a/src/netclass.c b/src/netclass.c
index d13d3f0..54c6d6e 100644
--- a/src/netclass.c
+++ b/src/netclass.c
@@ -31,16 +31,16 @@ static char *netclass_names[10] =
 
 static Coord clearances[10][10] =
   {
-    {MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2)}
+    {MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3)}
   };
